### PR TITLE
Add aux to ResponseItem for API 1.22

### PR DESCRIPTION
**Pull request by marcust**
As mentioned in Issue #437

---
This change is [Review on Reviewable](https://reviewable.io/reviews/docker-java/docker-java/442)

### CODE REVIEW

File: src/main/java/com/github/dockerjava/api/model/ResponseItem.java [hunk 1]
```diff
@@ -94,6 +95,11 @@ public String getError() {
         return error;
     }
 
+    @CheckForNull
+    public AuxDetail getAux() {
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/model/ResponseItem.java]: please set since tags

**Comment by marcuslinke** [comment]
@marcust Thanks. Will merge after v3.0.0 is released.

**Comment by KostyaSha** [comment]
@marcuslinke as soon as you end your refactorings i plan resolve #404 with reviewing newer API. Let me know when you end.

File: src/main/java/com/github/dockerjava/api/model/ResponseItem.java [hunk 2]
```diff
@@ -165,6 +178,40 @@ public String toString() {
         }
     }
 
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    public static class AuxDetail implements Serializable {
+        private static final long serialVersionUID = -9136704865403084084L;
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/model/ResponseItem.java]: https://github.com/jenkinsci/credentials-plugin/blob/master/CONTRIBUTING.md#resulting-from-long-experience about `1L`

**Comment by KostyaSha** [comment]
I will pick this into 1.22 branch. I think i we will keep it open until all tests will be added before real master merge.

**Comment by KostyaSha** [comment]
Can't reproduce "aux" field on 1.10.2 daemon...

**Comment by KostyaSha** [comment]
maybe it only on secured daemons https://github.com/docker/docker/blob/2892de760f412360c5effef9cfc346f2d92eaa4c/pkg/progress/progress.go#L19-L21

**Comment by KostyaSha** [comment]
@marcust could you provide some dump of this pull response?

**Comment by KostyaSha** [comment]
Merged, but json sample would be welcome!

**Pull request closed without merging**